server/controllers/payroll/multiplePayrollIndice/index.js

```diff
@@ -120,8 +120,11 @@ async function lookUp(options) {
   // let get rubric values for each employee
   const stagePaymentsParams = employeeUuid ? [payConfigId, db.bid(employeeUuid)] : [payConfigId];
   const stagePayments = await db.exec(stagePaymentIndiceSql, stagePaymentsParams);
+
   stagePayments.forEach(stagePay => {
-    employeesMap[stagePay.employee_uuid].rubrics.push(stagePay);
+    if (employeesMap[stagePay.employee_uuid]) {
```

Comment:
This will work around the crash, but won't the employee remain configured forever in the database? 

Is there some way to remove the employee id from the stage payments table?

Reply:
> This will work around the crash, but won't the employee remain configured forever in the database?
> 
> Is there some way to remove the employee id from the stage payments table?

To resolve this problem, I suggest doing a cleaning when we define the pay envelope